Python/Module.cpp

```diff
@@ -781,6 +797,8 @@ PyMODINIT_FUNC initPyHSPlasma() {
     PyModule_AddObject(module, "plConvexIsect", Init_pyConvexIsect_Type());
     PyModule_AddObject(module, "plSoundMsg", Init_pySoundMsg_Type());
     PyModule_AddObject(module, "plSwimMsg", Init_pySwimMsg_Type());
+    PyModule_AddObject(module, "plCameraMsg", Init_pyCameraMsg_Type());
+    PyModule_AddObject(module, "plAnimPath", Init_pyAnimPath_Type());
```

Comment:
The reason why these are listed above the last two types in this section is because they are in the wrong part of the list. Future work will be to move them into the correct section.